packages/notifications/demo/stories/ToastStory.tsx

```diff
@@ -20,7 +20,7 @@ import {
   MediaItem,
   Menu,
   Trigger
-} from '@zendeskgarden/react-dropdowns';
+} from '@zendeskgarden/react-dropdowns.legacy';
```

Comment:
This story should be using the non-legacy `Menu`.

Reply:
Yep, I realized that last night after putting up the PR. On it now :)